modules/ImportUtilities.js

```diff
@@ -138,11 +139,12 @@ class ImportUtilities {
         return graph;
     }
 
-    static prepareDataset(document, config, web3) {
+    static prepareDataset(originalDocument, config, web3) {
+        const document = originalDocument; // todo add otJsonService
```

Comment:
Do we still need this todo comment here?

Reply:
This is a placeholder for new sorting method which should be implemented in the next sprint